exec/java-exec/src/main/java/org/apache/drill/exec/physical/rowSet/AbstractRowSet.java

```diff
@@ -41,13 +38,19 @@ public AbstractRowSet(VectorContainer container, TupleMetadata schema) {
   }
 
   @Override
-  public VectorAccessible vectorAccessible() { return container(); }
+  public VectorAccessible vectorAccessible() {
```

Comment:
Why change formatting? Drill has long used single-line methods for conciseness. Is this an artifact of some code formatter? I didn't see discussion on the dev list about changing our coding standards...

Reply:
I think we should enforce such checkstyle. Taking into account that most of IDE can compress such methods automatically plus many other projects follow the same pattern. Personally, I was following such code style as well. @vvysotskyi could you please create Jira?

Reply:
I have created https://issues.apache.org/jira/browse/DRILL-7352 for this. I'll populate it with more rules soon.